app/javascript/src/Settings/components/Common/SelectGroup.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import * as React from 'react'
+import { useState } from 'react'
+import { FormGroup, Select, SelectOption } from '@patternfly/react-core'
+import type { FormSettingsProps, FormCatalogProps } from 'Settings/types'
+
+type Props = FormSettingsProps & FormCatalogProps
+
+const findInObject = (obj: { [key: string]: string }) => (key: string): string =>
```

Comment:
I think this is too much, it is equivalent as doing `catalog[key]` and delegate the default value to the select.

Reply:
OK if the array option is not suitable, then I propose using underscore's `each` and `map` that can iterate over objects. They can be installed independently (rather than the whole library) and it would improve reading IMO

Reply:
Or we create our own util method 🤷‍♂

Reply:
I don't think it's needed anymore, I got rid of that fn and made it easier to understand, I wasn't aware of passing an object with`toString` fn as a `value` in the past. wdyt?